src/main/java/hudson/plugins/sauce_ondemand/SauceOnDemandBuildWrapper.java

```diff
@@ -202,7 +205,8 @@ public SauceOnDemandBuildWrapper(
             boolean launchSauceConnectOnSlave,
             boolean useOldSauceConnect,
             boolean verboseLogging,
-            boolean useLatestVersion
+            boolean useLatestVersion,
+            boolean useGeneratedTunnelIdentifier
```

Comment:
No.  There would be no harm.

From: Alexey <notifications@github.com<mailto:notifications@github.com>>
Reply-To: jenkinsci/sauce-ondemand-plugin <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Monday, July 13, 2015 at 11:24 AM
To: jenkinsci/sauce-ondemand-plugin <sauce-ondemand-plugin@noreply.github.com<mailto:sauce-ondemand-plugin@noreply.github.com>>
Cc: Jessica Agte <jagte@shutterfly.com<mailto:jagte@shutterfly.com>>
Subject: Re: [sauce-ondemand-plugin] Updating newest Sauce OnDemand Plugin with auto tunnel generation. (#8)


In src/main/java/hudson/plugins/sauce_ondemand/SauceOnDemandBuildWrapper.java<https://github.com/jenkinsci/sauce-ondemand-plugin/pull/8#discussion_r34494228>:

> @@ -202,7 +205,8 @@ public SauceOnDemandBuildWrapper(
>              boolean launchSauceConnectOnSlave,
>              boolean useOldSauceConnect,
>              boolean verboseLogging,
> -            boolean useLatestVersion
> +            boolean useLatestVersion,
> +            boolean useGeneratedTunnelIdentifier


is there any harm in always using the generated Tunnel Identifier rather than adding a Boolean configuration option?

—
Reply to this email directly or view it on GitHub<https://github.com/jenkinsci/sauce-ondemand-plugin/pull/8/files#r34494228>.